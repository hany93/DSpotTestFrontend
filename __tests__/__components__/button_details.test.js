import ButtonDetails from '@components/button/button_details'
import { render, screen } from '@testing-library/react'

describe('ButtonDetails Component', () => {
	it('Render button details', () => {
		render(<ButtonDetails />)
		expect(screen.getByRole('link').getAttribute('href')).toBe('/')
	})
})
