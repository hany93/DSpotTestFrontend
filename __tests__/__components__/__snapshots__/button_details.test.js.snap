// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonDetails Component Test Snapshot 1`] = `
<div>
  <button
    class="profileslist__button"
  >
    <a
      href="/profiles_details/1"
    >
      Details
    </a>
  </button>
</div>
`;
